CONTRIBUTING.md

```diff
@@ -98,11 +98,13 @@ select the `eclipse.importorder` file as the import order config file.
 
 * Clone the repository: `git clone https://github.com/quarkusio/quarkus.git`
 * Navigate to the directory: `cd quarkus`
+* set Maven heap to 1.5GB `export MAVEN_OPTS="-Xmx1563m"`
```

Comment:
Maybe it should say `to at least 1.5GB`?

Reply:
I'd prefer we update the number instead of hiding behind a "at least" comment.

Reply:
Sure, that's fine with me